.circleci/config.yml

```diff
@@ -56,6 +55,101 @@ jobs:
 
 
 
+  build-linux-release:
+    <<: *defaults
+    docker:
+      - image: hyperledger/iroha-docker-develop
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - build-cache-release-{{ arch }}-{{ .Branch }}-
+            - build-cache-release-{{ arch }}-
+          paths:
+            - ~/.ccache
+      - run:
+          name: cmake
+          command: >
+            cmake \
+              -DCOVERAGE=OFF \
+              -DTESTING=OFF \
+              -H$IROHA_HOME \
+              -B$IROHA_BUILD \
+              -DCMAKE_BUILD_TYPE=Release \
+              -DPACKAGE_DEB=ON \
+              -DPACKAGE_TGZ=ON
+      - run:
+          name: make and package
+          command: |
+            cmake --build $IROHA_BUILD --target package -- -j4
+      - save_cache:
+          key: build-cache-release-{{ arch }}-{{ .Branch }}-{{ epoch }}
+          paths:
+            - ~/.ccache
+      - run:
+          name: copy deb file
+          command: |
+            cd $IROHA_BUILD
+            cp *.deb ./iroha.deb
```

Comment:
what if `*.deb` matches on >2 files?

Reply:
On the CI we can guarantee that there will be only one deb, and one tar.gz file.